ci-operator/step-registry/gather-network/gather-network-commands.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Running gather network..."
+mkdir -p ${ARTIFACT_DIR}/network
+oc --insecure-skip-tls-verify adm must-gather --dest-dir ${ARTIFACT_DIR}/network -- network > ${ARTIFACT_DIR}/network/network.log
```

Comment:
How does this work if you provide a command with `-- network` but not an `--image`?

Reply:
re working all of this